net/davix/src/TDavixFile.cxx

```diff
@@ -535,8 +535,9 @@ void TDavixFileInternal::parseConfig()
    // WLCG Bearer tokens check
    std::string prefix = "Bearer ";
    auto token = DiscoverToken();
-   if (token.c_str()) {
+   if (!token.empty()) {
       // header: "Authorization: Bearer mytoken"
+      R__LOG_INFO(TDavixLogChannel()) << "Using Bearer token: " << token;
```

Comment:
I'd remove this - this can easily become a security issue, depending on where logs end up. If you want to keep it then shorten it to
```suggestion
      R__LOG_INFO(TDavixLogChannel()) << "Using Bearer token starting with: " << token.substr(0, 3);
```